### ISSUE TITLE
When after running Ctrl+F5 ,the debugger is unable to exit and the debugger is unable to be called by F5

### ISSUE THREAD

**Issue by lyp694096417** [Microsoft/vscode#38371]
- VSCode Version: Code 1.18.0 (dcee2202709a4f223185514b9275aa4229841aa7, 2017-11-08T12:44:48.751Z)
- OS Version: Linux x64 4.12.0-041200-generic
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.11.0
code-gnu-global|aus|0.2.2
autofoldyeah|bob|1.0.6
dakara-foldplus|dak|1.1.0
MagicPython|mag|1.0.12
vscode-clang|mit|0.2.2
python|ms-|0.8.0
cpptools|ms-|0.14.2
vs-keybindings|ms-|0.1.7
vscode-icons|rob|7.18.1
markdown-preview-enhanced|shd|0.3.0
open-in-browser|tec|1.1.0
python|tht|0.2.3
pdf|tom|0.2.0
markdown-all-in-one|yzh|0.11.1

---

Steps to Reproduce:

1.
2.

Reproduces without extensions: Yes/No

**Comment by Balky79** [comment]
I'm experiencing the same issue.

I'm using VS code for python development. With the 1.18.0 update, I had to remove original "python" extension and install the new one published by Microsoft (0.8.0) in first place to make Code able to work with it. 

I'm seeing significant slow down in this version.

Then it does not exit the debug mode, and the only way to get it running again is to close Code and reopen it, which makes it terrible. 

On the rare occasion when I managed to re-run the same code twice, I've noticed that in this version it did not clear Debug console before re-running, which is weird. 

I'm on Windows 10.
